tests/integration/security/util/framework.go

```diff
@@ -85,6 +82,26 @@ func EchoConfig(name string, ns namespace.Instance, headless bool, annos echo.An
 				Protocol: protocol.GRPC,
 			},
 		},
+		// Workload Ports needed by TestPassThroughFilterChain
+		// The port 8085,8086,8087,8088 will be defined only in the workload and not in the k8s service.
+		WorkloadOnlyPorts: []echo.WorkloadPort{
```

Comment:
Hi @stevenctl, this is to address the comment in https://github.com/istio/istio/pull/28502